### PR TITLE
faq

### PR DESCRIPTION
**Pull request by devanshi-code**
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)

### Any relevant logs, error output, etc?
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

### Any other comments?
....

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**Ans:** Username:**admin1**
+ 
+ Password:**kep1**
+     
+
+
+**Q5.** Why do the credentials provided in Q3 and Q4 are not working?
+
+**Ans:** During login, the script checks the users’ table on the database to see if the provided login credentials are correct. Your database might be empty, so this will fail. Therefore, you must import filled_kidney_exchange_data.sql from the [manual setup](https://www.notion.so/Project-setup-Public-1a647ed8515c485f99f38e717acfa61b) document for the credential provided in Q3, Q4 to work.
```
Comment (siv2r) [inline docs/FAQ.md:30]: bound the filled_kidney_exchange_data.sql with backticks so, it looks like this `filled_kidney_exchange_data.sql`

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# **<center>FAQ'S</center>**
+
+**Q1.** Who all can use this website?
```
Comment (siv2r) [inline docs/FAQ.md:3]: Can make all the question bold? Also, make their font larger than the answer

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+**Q6.** I am unable to find the hospital registration form on the website. What should I do?
+
+**Ans:** You might have logged in using a transplant coordinator’s credentials (provided in Q3). Some features are invisible to the transplant coordinator since they are end-users and not a part of the development team. Login using admin credentials (provided in Q4). You can view the hospital registration form under register in the navbar.
+
+**Q7.** Global match (inside the match section in the nav bar) is not working.?
+
+**Ans:** Run this command in root folder:
+        sudo chmod 777 global_match
+
```
Comment (siv2r) [inline docs/FAQ.md:40]: add backticks for this sudo command too

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+**Ans:** You might have logged in using a transplant coordinator’s credentials (provided in Q3). Some features are invisible to the transplant coordinator since they are end-users and not a part of the development team. Login using admin credentials (provided in Q4). You can view the hospital registration form under register in the navbar.
+
+**Q7.** Global match (inside the match section in the nav bar) is not working.?
+
+**Ans:** Run this command in root folder:
+        sudo chmod 777 global_match
+       
+ This command sets the write permission as everyone for the global_match directory. Now the python script can write files/folders to the global_match directory. Hence, the Global match option in the website will work.
+
+**Q8.**  I am getting a **<ins>hospital not registered</ins>** error when I try to sign up. Why is this happening?
```
Comment (siv2r) [inline docs/FAQ.md:43]: Is it possible to make the 'hospital not registered' in red color

**Review by siv2r — CHANGES_REQUESTED** [review]

File: docs/FAQ.md (hunk 1 above)
Comment (devanshi-code) [reply docs/FAQ.md:30]: okay i will do this change till morning

File: docs/FAQ.md (hunk 2 above)
Comment (devanshi-code) [reply docs/FAQ.md:3]: I can make it bold , and will try to make font larger but at that time I tried it wasn't happening

File: docs/FAQ.md (hunk 3 above)
Comment (devanshi-code) [reply docs/FAQ.md:40]: backticks refers to adding a link or what?

File: docs/FAQ.md (hunk 4 above)
Comment (devanshi-code) [reply docs/FAQ.md:43]: No it isn't possible I have tried to search in many places but couldn't get solution for this

File: docs/FAQ.md (hunk 1 above)
Comment (siv2r) [reply docs/FAQ.md:30]: Sure no worries :)

File: docs/FAQ.md (hunk 2 above)
Comment (siv2r) [reply docs/FAQ.md:3]: if use # it becomes a heading so, try using a heading with the smallest size. I know ### is small but not sure if #### exists

File: docs/FAQ.md (hunk 3 above)
Comment (siv2r) [reply docs/FAQ.md:40]: ` this symbol is backick

File: docs/FAQ.md (hunk 4 above)
Comment (siv2r) [reply docs/FAQ.md:43]: Okay, then just backtick for this too

File: docs/FAQ.md (hunk 3 above)
Comment (siv2r) [reply docs/FAQ.md:40]: It is just to mention it is a code

File: docs/FAQ.md (hunk 2 above)
Comment (devanshi-code) [reply docs/FAQ.md:3]: Hey can u explain this again heading refers to faq or what , I have done all the other changes so once this one is done too then I will make pr

**Comment by siv2r** [comment]
Great work :tada: :tada:

**Pull request merged**